app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -25,6 +25,26 @@
       <!-- Only is mounted where there are attributes -->
       <SummaryTab
         v-if="activeDatasetTab === 0 && attributes"
+        :dataset-id="datasetId"
+        is-viz-saving-prompt-visible="isVizSavingPromptVisible"
```

Comment:
Is this working? You miss the colon

Reply:
Yes, it's working because this prop is unnecessary.